src/components/Filters.js

```diff
@@ -1,4 +1,15 @@
-const Filters = () => {
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+
+const Filters = (props) => {
+  const filters = document.querySelectorAll('.sidebar__link');
+  const pags = document.querySelectorAll('.pag__link');
```

Comment:
Very often it's not a good idea to touch DOM 'by hand` in React. Think a little maybe you can avoid that.